### DISCUSSION TITLE
Porblems with QuoteSource.get_source('Yahoo')

### DISCUSSION THREAD

**Discussion by danigarod** [dodid/minitrade#4] [General]
Hi, team; I am trying to use the QuantSource method to retrieve data from Yahoo, but I am getting this error. Do you have any idea of the problem?

Exception has occurred: ImportError
cannot import name 'config' from 'minitrade.utils.config' (C:\Users\garcd3\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\minitrade\utils\config.py)
  File "C:\Users\garcd3\Trade\Old\Minitrade.py", line 93, in <module>
    yahoo = QuoteSource.get_source('Yahoo')
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ImportError: cannot import name 'config' from 'minitrade.utils.config' (C:\Users\garcd3\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\minitrade\utils\config.py)

**Comment by dodid** [comment]
This should have been fixed in the latest version. closing.
